tests/base/src/test/java/org/wildfly/security/http/HttpAuthenticatorTest.java

```diff
@@ -255,5 +256,24 @@ public void testUsingSecurityProviderServerMechanismWithDigestMD5() throws Excep
 
         authenticateWithDigestMD5();
     }
-
```

Comment:
@keshav-redhat Just realized new test is added in HttpAuthenticatorTest. But we do not use any `HttpAuthenticator` in this new test. Can you please create new test directory for `bearer` in http directory and put this test there?

Reply:
@keshav-redhat I think this still needs to be updated.